app/react/settings/users/UserForm.jsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import PropTypes from "prop-types";
+import InlineLoading from "react/common/ui/InlineLoading";
+
+class UserForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: "",
+            password: "",
+            realname: "",
+            role: "Select a role"
+        };
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.renderRoles = this.renderRoles.bind(this);
+    }
```

Comment:
I think it might be a useful convention to always order methods as
```
class Blah {
  constructor()
  render()
  everythingElse()
  ...
}
```
So that we can immediately see the rendered result for a component. I'm not sure where lifecycle hooks fit in that, I think airbnb might already have rules about it?
What do you think? Alternative might be `render` is always last

Reply:
Robbie to update if there is a standard convention. I will refactor

Reply:
https://github.com/airbnb/javascript/tree/master/react#ordering

Reply:
https://github.com/airbnb/javascript/tree/master/react#ordering <- turns out this is the answer